content/src/service/synchronization/EventDeployer.ts

```diff
@@ -170,7 +170,7 @@ export class EventDeployer {
 
   private buildDeploymentExecution(
     deploymentEvent: DeploymentWithAuditInfo,
-    execution: () => Promise<void>
+    execution: () => Promise<Timestamp | ErrorList>
```

Comment:
Is this change necessary?

Reply:
That was the way that I found to check whether the deployment was successful or not inside the `wrapDeployment` method, do you have any idea on how to avoid that?

Reply:
Oh, I see. Makes sense 😄